preload/vperm_filestatgates.c

```diff
@@ -256,6 +255,23 @@ int __fxstatat64_gate(int *result_errno_ptr,
 	return(res);
 }
 
+int stat64_gate(int *result_errno_ptr,
+	int (*real_stat64_ptr)(const char *file_name, struct stat64 *buf64),
```

Comment:
Should this have same format as others, i.e.: `real___stat64_ptr` ?

Reply:
This is the correct format, some function names have __ in the beginning of the name which leads to ___ but since this doesn't the naming is correct.